contracts/OysterPearl.sol

```diff
@@ -129,6 +130,9 @@ contract OysterPearl {
 
     // Prevents accidental lockout
     require(msg.value == 10 ether);
+    
+    // refunds security
+    director.transfer(10 ether);
```

Comment:
So just so I'm clear on what's going on here.  Originally the director had to lose 10 ETH to lock itself out.  And what you are doing here is refunding the director back their money?  

I don't have any strong feelings giving the director back their money, but I think the point of it was to make it a serious decision to lock out the director.